### PR TITLE
Introduce basic modules for Flink CDC 3.0

### PR DESCRIPTION
**Pull request by leonardBang**
*  Introduce basic modules for Flink CDC 3.0
* Bump version to 3.0-SNAPSHOT
* Fix mysql e2e failure due to docker file mapping

### CODE REVIEW

File: flink-cdc-connect/cdc-pipeline-connectors/pom.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>flink-cdc-connect</artifactId>
+        <groupId>com.ververica</groupId>
+        <version>3.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>cdc-pipeline-connectors</artifactId>
```
Comment (PatrickRen) [inline flink-cdc-connect/cdc-pipeline-connectors/pom.xml:26]: Maybe `flink-cdc-pipeline-connectors`?

**Review by PatrickRen — COMMENTED** [review]
@leonardBang Thanks for kicking off the new major version! Just one comment about the module name of connectors

File: flink-cdc-connect/cdc-pipeline-connectors/pom.xml (hunk 1 above)
Comment (ruanhang1993) [reply flink-cdc-connect/cdc-pipeline-connectors/pom.xml:26]: Besides this, `cdc-source-connectors` / `cdc-pipeline-e2e-tests` / `cdc-source-e2e-tests` need to change.

**Comment by leonardBang** [comment]
Thanks @PatrickRen @ruanhang1993  for the review, I've addressed your comments

**Review by PatrickRen — APPROVED** [review]
@leonardBang Thanks for the update! LGTM.

**Pull request merged**